### DISCUSSION TITLE
Unable to open SU2 restart file

### DISCUSSION THREAD

**Discussion by Tushma** [su2code/SU2#1481] [Q&A]
Hi,

I'm fairly new to SU2 so apologies if this is an obvious question. I'm trying to run an unsteady simulation based on the APU_turbocharger, I've implemented the time-dependent simulation options based on the unsteady test cases also available. Whenever I run my simulation in parallel with 32 partitions I get the error

Error in "void CSolver::Read_SU2_Restart_Binary(CGeometry*, const CConfig*, std::string)":
----------------------------------------------------------------------------------------------------------
Unable to open SU2 restart file restart_flow_0_00100.dat

I have attached the modified config file from the APU_turbocharger test case, can anyone help with this issue?

Thank you in advance
[APU.txt](https://github.com/su2code/SU2/files/7829438/APU.txt)
e

**Comment by Thx666** [comment]
Did you fix this?
I would appreciate if you help me out.
![image](https://user-images.githubusercontent.com/93650463/221860515-f697f96f-140a-4e2d-9edb-49f803a306a7.png)

**Reply by WallyMaier** [reply]
Please be sure solution.dat exists in your folder.  If not, rename restart.dat to solution.dat

**Comment by WallyMaier** [comment]
@Tushma are your running a restart solution? If so, please be sure the restart file exist in the directory.   Also is the issue only present for 32 cores?

**Reply by Thx666** [reply]
Got it,
if there are only two files: .cfg and .su2, 
command in .cfg should be like this:
![image](https://user-images.githubusercontent.com/93650463/222067125-aaaa9503-6909-4bab-b816-c089ee289e9a.png)
then we can run a restart solution by "YES", If not, rename restart.dat to solution.dat 
![image](https://user-images.githubusercontent.com/93650463/222068356-29508a3a-c9d6-486f-96a0-c27f9b68cad7.png)

**Comment by bigfooted** [comment]
Note the following config options:
```
% SU2 Output file for restart
RESTART_FILENAME= restart.dat
% SU2 Input file for restart
SOLUTION_FILENAME= solution.dat
%
```

They have different names by default to prevent accidental overwrites.
